pasqal_cloud/client.py

```diff
@@ -206,7 +206,7 @@ def get_batch_jobs(self, batch_id: str) -> list[Dict[str, Any]]:
             # the same order as they do in the GET /batches/{id} response
         )
 
-    @retry_http_error(max_retries=2)
+    @retry_http_error(max_retries=2, retry_exceptions=(requests.ConnectionError,))
```

Comment:
So they are failing due to network issues or synchronicity issues? I thought the test was running faster than an updated resource and this was the failure. We _know_ it's due to a network issue?

Should we bump it to 5? If something fails due to a network issues, it could well fail again immediately if the expected first delay is 1 second